packages/atomic/src/pages/examples/commerce-website/listing-pants.html

```diff
@@ -16,8 +16,6 @@
         const commerceInterface = document.querySelector('atomic-commerce-interface');
 
         await commerceInterface.initialize(commerceEngineConfig);
-
-        commerceInterface.executeFirstSearch();
```

Comment:
Added unintentionally included in my previous PR

Reply:
?  wdym: should we keep or remove the line? Not sure to understand

Reply:
With this line we end up with both a product listing query and a search query instead of just the former